File: MobileHenHo/lib/Models/Feature.dart
import 'package:smart_select/smart_select.dart';
import 'package:tinder_clone/Models/FeatureDetail.dart';

class Feature {
  int id;
  String name;
  double weightRate;
  bool isCalculated;
  bool isSearchFeature;
  List<FeatureDetail> featureDetails = [];
  void setFeatureDetails(dynamic response) {
    for (var item in response) {
      FeatureDetail featureDetail = new FeatureDetail();
      featureDetail.id = item['id'];
      featureDetail.featureId = item['featureId'];
      featureDetail.content = item['content'];
      featureDetail.weight = item['weight'];
      featureDetails.add(featureDetail);
    }
  }
}

File: MakeFriendSolution/MakeFriendSolution/Models/Feature.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MakeFriendSolution.Models
{
    public class Feature
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double WeightRate { get; set; }
        public bool IsCalculated { get; set; }
        public bool IsSearchFeature { get; set; }

        public ICollection<UserFeature> UserFeatures { get; set; }
        public ICollection<SearchFeature> SearchFeatures { get; set; }
        public ICollection<FeatureDetail> FeatureDetails { get; set; }
    }
}